src/Test/contconic.jl

```diff
@@ -230,9 +230,9 @@ function lin3test(model::MOI.ModelLike, config::TestConfig)
     x = MOI.addvariable!(model)
 
     @test MOI.canaddconstraint(model, MOI.VectorAffineFunction{Float64}, MOI.Nonnegatives)
-    MOI.addconstraint!(model, MOI.VectorAffineFunction([1],[x],[1.0],[-1.0]), MOI.Nonnegatives(1))
+    MOI.addconstraint!(model, MOI.VectorAffineFunction(MOI.VectorAffineTerm.([1], MOI.ScalarAffineTerm.([1.0], [x])),[-1.0]), MOI.Nonnegatives(1))
```

Comment:
Unnecessary broadcast? Why not:
```julia
MOI.VectorAffineTerm(1, MOI.ScalarAffineTerm(1.0, x))
```

Reply:
One other think that could be changed would be thinks like
```julia
MOI.ScalarAffineTerm.(one(3), [x, y, z])
```
into
```julia
MOI.ScalarAffineTerm.(1.0, [x, y, z])
```